server/routes/index.ts

```diff
@@ -3,13 +3,15 @@ import type { Services } from '../services'
 import ContactsRoutes from './contacts/routes'
 import SearchRoutes from './search/routes'
 import SEARCH_PRISONER_URL from './urls'
+import HomeRoutes from './home/routes'
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export default function routes({ auditService, prisonerSearchService }: Services): Router {
   const router = Router({ mergeParams: true })
 
-  router.use('/', ContactsRoutes(auditService))
   router.use(SEARCH_PRISONER_URL, SearchRoutes(auditService, prisonerSearchService))
+  router.use('/contacts', ContactsRoutes(auditService))
+  router.use('/', HomeRoutes(auditService))
 
   return router
```

Comment:
I'm going to remove this SEARCH_PRISONER_URL constant - prefer to see the actual URL portions as you have it here.